testDll/op2ext.test.cpp

```diff
@@ -6,6 +6,10 @@
 #include <windows.h>
 
 
+// MSVC C4068 (unknown pragma) is issued on calls to #pragma GCC
+#pragma warning(disable: 4068)
```

Comment:
Ahh, we should maybe guard that pragma so it's only seen by Mingw, rather than disable MSVC warnings about it.